### DISCUSSION TITLE
[@types/three]

### DISCUSSION THREAD

**Discussion by ktedore** [DefinitelyTyped/DefinitelyTyped#58682] [Issues with a @types package] [answered]
three.js r135
GLTFExporter - Add onError parameter to .parse()

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7467038c60726c43ccfc0f084cd8d5d8f062d234/types/three/examples/jsm/exporters/GLTFExporter.d.ts#L18

[a comment by an automated account is not shown]

**Comment by joshuaellis** [comment, marked as answer]
Can you please put this issue in [`three-ts-types`](https://github.com/three-types/three-ts-types)?
